f3/granite.go

```diff
@@ -536,14 +547,19 @@ func (i *instance) beginConverge() {
 		}
 		justification = Justification{
 			Vote:      justificationPayload,
-			Signers:   signers,
+			Signers:   quorum.SignersBitfield(),
 			Signature: aggSignature,
 		}
 	} else if justification, ok = prevRoundState.committed.justifiedMessages[i.proposal.Head().CID]; ok {
 		//justificationPayload already assigned in the if statement
 	} else {
 		panic("beginConverge called but no evidence found")
 	}
+	ticket, err := i.vrf.MakeTicket(i.beacon, i.instanceID, i.round)
+	if err != nil {
+		i.log("creating VRF ticket: %v", err)
```

Comment:
Include the word "error"? This will be clearer once we introduce a proper logger.